controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3752,4 +3752,13 @@ OperationFailedRuntimeException capabilityAlreadyRegisteredInContext(String capa
 
     @Message(id = 504, value = "The operation %s is not defined for resource %s.")
     UnsupportedOperationException missingOperationForResource(String op, String address);
+
+    @Message(id = 505, value = "Unsuported deployment yaml file %s with attributes %s")
+    IllegalArgumentException unsupportedDeployment(String deployment, Set<String> attribues);
+
+    @Message(id = 506, value = "The yaml element \"%s\" and its sub-elements are ignored.")
```

Comment:
Dummy issue but can be space added at the end of message like:
`@Message(id = 506, value = "The yaml element \"%s\" and its sub-elements are ignored. ")`

to avoid: `ignored.Thus ignoring`  in:  
```
12:59:42,222 WARN  [org.jboss.as.controller.management-operation] (main) WFLYCTL0506: The yaml element "extension" and its sub-elements are ignored.Thus ignoring element "org.jboss.as.failure: {module: org.jboss.as.failure}".
```